rr_computer_vision/include/endline_detection.hpp

```diff
@@ -0,0 +1,42 @@
+/** @file endline_detection.hpp
+ * @author Waleed Ahmed (w29ahmed)
+ * @author Yuchi(Allan) Zhao
+ */
+
+#ifndef ENDLINE_DETECTION_HPP
+#define ENDLINE_DETECTION_HPP
+
+#include <ros/ros.h>
+#include <image_transport/image_transport.h>
+#include <cv_bridge/cv_bridge.h>
+#include <sensor_msgs/image_encodings.h>
+#include <opencv2/opencv.hpp>
+#include <std_srvs/Trigger.h>
+// #include "msg_srv_names.hpp"
```

Comment:
Please remove this if this is not used anywhere

Reply:
Had it here cuz wanted to put the supervisor service callbacks into that common namespace and reference them in here that way instead of hardcoded strings. But this is extra work and Ima just put it off tbh cuz who really cares